nette.ajax.js

```diff
@@ -329,7 +329,15 @@ $.nette.ext('validation', {
 
 		if (validate.url) {
 			// thx to @vrana
-			if (/:|^#/.test(analyze.form ? settings.url : analyze.el.attr('href'))) return false;
+			var urlToValidate = analyze.form ? settings.url : analyze.el.attr('href');
+			//Check if URL is absolute
```

Comment:
Please put space after `//` (it's easy to get this right first time by checking already existing codebase).